source/extensions/filters/common/rbac/utility.cc

```diff
@@ -11,6 +13,13 @@ RoleBasedAccessControlFilterStats generateStats(const std::string& prefix, Stats
   return {ALL_RBAC_FILTER_STATS(POOL_COUNTER_PREFIX(scope, final_prefix))};
 }
 
+std::string responseDetail(const std::string& policy_id) {
+  // Replace whitespaces in policy_id with '_' to avoid potential breaks in the access log.
```

Comment:
Expand potential breaks a bit with: inconsistent number of segments between log entries when the separator is space?